### ISSUE TITLE
In-country summary table in Piwik

### ISSUE THREAD

**Issue by MartinMaritim** [humanitarianresponse/site#139]
As discussed with @javedyousafzai this morning, we need to add in-country summary table in Piwik. Currently, as seen in the attached screenshot, we only have summary stats for all visits.
<img width="1202" alt="summary" src="https://cloud.githubusercontent.com/assets/3490890/10453050/fc0b88b0-71b1-11e5-933d-8a55d927bfe0.png">

**Comment by pcambra** [comment]
I don't think we can do this at the moment, we need to solve #138 first and then have piwik in its own container.

**Comment by pcambra** [comment]
What is missing here is a column called '% in country visits', and not a subtable per in-country.
